### ISSUE TITLE
Action selection style for Gantt charts

### ISSUE THREAD

**Issue by abhijit945** [cerner/carbon-graphs#5]
Once Event selection has been implemented for Gantt charts, we may want to consider changing the current circular selection pattern for Actions to match the rectangular shape used elsewhere in Gantt charts.

Further thoughts/discussion, especially for MAR, are needed.

**Comment by laurunning** [comment]
Propose that the selection match bar selection for consistency - ![image](https://user-images.githubusercontent.com/32143728/66942264-4d623180-f00e-11e9-9b4b-cd39c38ec1df.png)
Related to centering carbon chunks issue: https://github.com/cerner/carbon-graphs/issues/100

**Comment by laurenbock** [comment]
Yes - enhancement, doable, follow rectangular pattern as in bars.

**Comment by laurenbock** [comment]
Note: this one will cover https://github.com/cerner/carbon-graphs/issues/115#issuecomment-568024678 so it will be close. The surrounding selection should wrap around the item that was clicked wherever it is placed in the row.

**Comment by mrw100** [comment]
Is there any way we can get this prioritized? We are planning a release in March and there are concerns that there will be confusion as to what is selected due to the selection bug mentioned in #115.

![event-selection-issue](https://user-images.githubusercontent.com/1413706/73953815-8f1d2d80-48c6-11ea-811e-c1dc950d07df.png)

We are planning on displaying events below the blue background bar, so it will be possible for an event below the background bar to look selected even though the event in the middle of the bar is what's actually selected.

**Comment by laurenbock** [comment]
@mrw100 we split this into two separate issues again. A bug for the alignment and this one for the shape of the selection surround. Here is the bug: https://github.com/cerner/carbon-graphs/issues/170

**Comment by pranav300** [comment]
@mrw100 
Can you provide a link for this Gantt chart?

**Comment by mrw100** [comment]
@pranav300 https://pages.github.cerner.com/medication-record/mar-js/raw/tests/mar-js/main-content/standalone
